projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/notification/DefaultEmailNotification.java

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
This is not new code. We renamed this class from EmailNotification.

Reply:
Add this a comment as well